src/main/java/com/google/api/generator/engine/escaper/CommentEscaper.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.escaper;
+
+import com.google.common.escape.Escaper;
+import com.google.common.escape.Escapers;
+
+public class CommentEscaper {
+  private static class SpecialEscaper extends Escaper {
+    private static final Escaper escaper =
+        Escapers.builder()
+            .addEscape('\t', "\\t")
+            .addEscape('\b', "\\b")
+            .addEscape('\r', "\\r")
+            .addEscape('\f', "\\f")
+            .addEscape('\n', "\\n")
+            .addEscape('\\', "\\\\")
+            .build();
+
+    private SpecialEscaper() {}
+
+    @Override
+    public String escape(String sourceString) {
+      // Replace unexpected block end `*/` with `* /`.
+      return escaper.escape(sourceString.replace("*/", "* /"));
```

Comment:
Could you please provide more context for this logic? I'm not sure that replacing `*/` with `* /` is the appropriate approach because this edits the comment.

Reply:
sure, this covers the case that we have `*/` in the middle of comment string, that will break Block/JavaDoc comment since it's an unexpected end.
For example (unit test also shows this case):
```
/** this is a block comment with unexpected end */ we should replace it. */
```

Reply:
A cursory search shows that [HTML escaping](https://stackoverflow.com/questions/631817/how-to-quote-in-javadocs) is one possible approach, which I think we should take. Alternatively, would it be valid if we escaped the `*` with a backslash? Adding a space essentially changes the comment's contents, which is undesirable.

If we pick the HTML escaper route (and I think we should), feel free to add a TODO and leave it for the next PR (below).

As a side note (not related to this PR), I've since noticed that the baseline file intentionally performs HTML escaping on special characters such as `<` and `>`. (We briefly talked about these chars in the original comment PR and thought that rendering them as-is was the intended behavior.) As a separate task (which I've added to the work tracker sheet), could you please add an HTML escaper for JavaDocComment in a separate PR?

Reply:
Sounds cool, that explains why we have `&lt;&gt;&amp;` etc in the comment, which is for rendering `<>` correctly in browser. Thanks for the advice! Removed the html part in tests and added a TODO for the HTML escaper, and `*/` test case will be added by that implementation.